aws-ecs/01-variables.tf

```diff
@@ -78,10 +91,16 @@ variable "rds_port" {
   default     = "5432"
 }
 
+variable "rds_backup_retention_period" {
+  description = "Pvault RDS backup_retention_period"
+  type        = number
+  default     = 30
+}
+
 variable "pvault_image" {
   description = "Pvault image:tag public image"
   type        = string
-  default     = "public.ecr.aws/s4s5s6q8/pvault-server:1.2.2"
+  default     = "piiano/pvault-server:1.3.1"
```

Comment:
nit: I suggest splitting this variable to "repository" and "tag" (like with helm)

Reply:
done